Automattic/wp-calypso
# GuidedTours: Remove test from signup & remove redundant code

This will end the abtest for the `main` Guided Tour. It also removes a bunch of obselete, or soon to be made obselete code. This includes the showGuidedTour action, and the `?tour=[x]` query arg interception.


To test:
1. Verify the purchase flow still works, and no tour is seen
2. Verify that the `tour` query arg has no effect

/cc @lsinger @mcsf

Test live: https://calypso.live/?branch=update/remove-main-guided-tour-from-signup
